File: src/main/java/brzzzn/fabadditions/FabAdditions.java
package brzzzn.fabadditions;

import brzzzn.fabadditions.registry.FabBlocks;
import brzzzn.fabadditions.registry.FabItems;
import net.fabricmc.api.ModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class FabAdditions implements ModInitializer {

	public static final String ID = "fabadditions";

	public static final Logger LOGGER = LoggerFactory.getLogger("fabadditions");

	@Override
	public void onInitialize()
	{
		FabItems.registerModItems();
		FabBlocks.registerModBlocks();
	}
}

File: src/main/kotlin/brzzzn/fabadditions/FabAdditions.kt
package brzzzn.fabadditions

import brzzzn.fabadditions.registry.*
import net.fabricmc.api.ModInitializer
import org.slf4j.LoggerFactory

class FabAdditions : ModInitializer {
    override fun onInitialize()
    {
        FabCommandRegistry.registerCommands()
        FabItemsRegistry.registerModItems()
        FabBlockRegistry.registerModBlocks()
        FabBlockEntityRegistry.registerModBlockEntities()
        FabEntityRegistry.registerModEntities()
    }

    companion object {
        const val ID = "fabadditions"

		val logger = LoggerFactory.getLogger("fabadditions")
    }
}